### PR TITLE
Updating a few tips with type fixes and some display issues.

### PR DESCRIPTION
**Pull request by asoffer**

### CODE REVIEW

File: _posts/2017-09-26-totw-1.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Tip of the Week #1: <code>string_view</code>"
```
Comment (manshreck) [inline _posts/2017-09-26-totw-1.md:2]: What's the display issue here? These are supposed to be displayed as monospaced

Comment (asoffer) [reply _posts/2017-09-26-totw-1.md:2]: This one is fine, the <> in StatusOr on 181 is causing problems:
https://abseil.io/tips/

Comment (manshreck) [reply _posts/2017-09-26-totw-1.md:2]: Ah, yes, does change the one in StatusOr to <code>StatusOr&lt;gt;</code> solve it ? I'd rather us remain consistent with code in titles.

Comment (asoffer) [reply _posts/2017-09-26-totw-1.md:2]: We're already inconsistent and I'd prefer the inconsistency to fixing things manually. Can we accept this automated change and work on fixing the automation as a follow-up?

Comment (manshreck) [reply _posts/2017-09-26-totw-1.md:2]: Sure

**Review by manshreck — APPROVED** [review]

**Pull request merged**